test-app/lib/Router.coffee

```diff
@@ -0,0 +1,22 @@
+Session.setDefault '___isPhantomjs___', false if Meteor.isClient
+
+Router.onAfterAction ->
+  Meteor.isReadyForSpiderable = true if @ready()
+
+Router.configure
+  notFoundTemplate: '_404'
+
+Router.plugin 'dataNotFound', 
+  notFoundTemplate: Router.options.notFoundTemplate
+
+if Meteor.isClient
+  Router.onBeforeAction -> 
+    if _.has @params.query, '___isRunningPhantomJS___'
+      Session.set '___isPhantomjs___', true
```

Comment:
I am going to remove this Session.set, since it is not required to demonstrate that Spiderable works, and also not the minimum required setup. (right?)

I believe that we have some complexity already, in setting things up, and people will use this example as a simple way to get started and to debug.